modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -47,14 +63,21 @@ scp /backup-path/to-restore.tar.gz your_username@remotehost.edu:/var/lib/gravity
 ----
 sudo gravity restore /var/lib/gravity/planet/share/to-restore.tar.gz
 ----
-
++
 You can pass the following optional parameters:
-
++
+====
 * `--follow`: outputs the backup job logs to stdout
 * `--timeout`: specifies the deadline for the backup job (e.g.: 30s, 5m, etc). Default is 20m.
 * `--log-file`: outputs logs to a file
+====
++
+Wait until the operation completes. This operation takes a few minutes. 
 
-Note that this operation takes a few minutes. 
+. Manually restore the following information: 
++
+* Runtimes
```

Comment:
You don't need to manually restore runtimes

Reply:
Removed - thank you!